modules/ml/src/svm.cpp

```diff
@@ -2261,6 +2261,17 @@ Ptr<SVM> SVM::create()
     return makePtr<SVMImpl>();
 }
 
+Ptr<SVM> SVM::load(const String& filename)
+{
+    FileStorage fs;
+    fs.open(filename, FileStorage::READ);
+
+    Ptr<SVM> svm = makePtr<SVMImpl>();
+
+    svm->read(fs.getFirstTopLevelNode());
```

Comment:
Thank you very much for potining out this issue.
I'm going to fix this issue asap. :+1: